### PR TITLE
docs: update Rust/Python migration guide for serialization

### PR DESCRIPTION
**Pull request by wyfo**

### CODE REVIEW

**Review by DariusIMP — APPROVED** [review]
LGTM

File: content/docs/migration_1.0/Python.md [hunk 1]
```diff
@@ -37,50 +37,68 @@ with zenoh.open(zenoh.Config()) as session:
...
+- Zenoh 1.0.0
+
+```rust
+sample = subscriber.recv()
+my_string = sample.payload.to_string()
+```
+
+You can look at a full set of examples in `examples/z_bytes.py`.
+
+### Serialization
```
Comment (oteffahi) [inline content/docs/migration_1.0/Python.md:64]: Same issue as on Rust: this is the only "sub-title", which makes the summary of the page look weird. Consider changing it.

```suggestion
## Serialization
```

File: content/docs/migration_1.0/Python.md [hunk 2]
```diff
@@ -37,50 +37,68 @@ with zenoh.open(zenoh.Config()) as session:
...
-import json
-publisher = session.declare_publisher("my/keyepxr", encoding=zenoh.Encoding.APPLICATION_JSON)
-publisher.put(json.dumps({"key", "value"}))  # default encoding from publisher `application/json`
+input = b"raw bytes"
+payload = ZBytes(input)
+output = payload.to_bytes()
 ```
 
-### (De)serialization
+`zenoh.ext` serialization doesn't pretend to cover every use cases, as it is just one available choice among other serialization format like JSON, Protobuf, CBOR, etc. In the end, Zenoh will just send and receive payload raw bytes independently of the serialization used.
```
Comment (oteffahi) [inline content/docs/migration_1.0/Python.md:74]:
```suggestion
`zenoh.ext` serialization doesn't pretend to cover all use cases, as it is just one available choice among other serialization formats like JSON, Protobuf, CBOR, etc. In the end, Zenoh will just send and receive payload raw bytes independently of the serialization used.  
```

**Review by oteffahi — CHANGES_REQUESTED** [review]
Some minor changes (nitpicks) for grammar and title levels.
Other than that it LGTM.

**Review by oteffahi — APPROVED** [review]

**Pull request merged**